addons/calendar/controllers/main.py

```diff
@@ -3,30 +3,22 @@
 
 import werkzeug
 
-from odoo.api import Environment
 import odoo.http as http
 
 from odoo.http import request
-from odoo import SUPERUSER_ID
-from odoo import registry as registry_get
```

Comment:
I would add the following explanation on the commit message (I let you rephrase it if you wish to keep it real).

Before we wanted to manage the multi-db thing.

Now we expect the users to work live on the same db. (That's the reason why in a multi-db environment, the database manager opens to force you choosing one). Everything is managed at the dispatcher side. It's not the role of the business code to think about the infrastructure. As for the actions buttons on the notification emails for example.

Reply:
I'm not sure to understand all this multi-db thing